spec/PhpSpec/Util/ExampleObjectUsingTrait.php

```diff
@@ -0,0 +1,6 @@
+<?php namespace spec\PhpSpec\Util;
```

Comment:
Is there a reason you've not inlined this into the spec? It's what we've done in other similar situations.